### ISSUE TITLE
Color Contrast is not meeting required ratio for text in bot of Microsoft Teams

### ISSUE THREAD

**Issue by MayurMundekar** [MicrosoftDocs/msteams-docs#4895]
We own the Azure Pipeline app for teams.
After running the command ""/azpipelines signin" it provide the command list. but for that ui color contrast is not meeting the required ration for text as per accessibility insights.
The color contrast should be **equal to or greater that 4.5:1**
 
**MAS Reference:**
[MAS 1.4.3 – Contrast (Minimum)](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B0BF582E1-6800-4E43-8947-DA015AA28B2B%7D&file=MAS%201.4.3%20%E2%80%93%20Contrast%20(Minimum).docx&wdLOR=c77A244B8-8386-4532-8ADC-E7D5BC75A52E&action=default&mobileredirect=true&cid=84581af4-d2b6-4d3c-bbe5-7ec467ca1ef0)

This is an accessibility bug.

In the below screenshot, the red exclamation marks are for reference.

![1882831_A11y_AzDev-azchatops_Help_Help_Contrast (Minimum)_AI4W](https://user-images.githubusercontent.com/83640993/146932500-542be392-f406-4159-a7bf-227a0956e661.png)
